x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_finding_flyout/vulnerability_finding_flyout.test.tsx

```diff
@@ -93,15 +93,15 @@ describe('<VulnerabilityFindingFlyout/>', () => {
     });
   });
 
-  it('should allow pagination with next', async () => {
+  it.skip('should allow pagination with next', async () => {
```

Comment:
Exactly, I skipped them because there's a plan to re-enable later once https://github.com/elastic/kibana/issues/168619 is resolved. I added a TODO comment on the skipping so we can know when to re-enable.